test/hist.jl

```diff
@@ -219,6 +219,21 @@ end
     @test (@inferred merge(histograms...)) == h
 end
 
+@testset "Histogram arithmetics" begin
+    h1 = fit(Histogram, rand(100), 0:0.1:1)
+    h2 = fit(Histogram, rand(200), 0:0.1:1)
+    h3 = fit(Histogram, rand(100), 0:0.2:1)
+    @test (h1 + h2).weights == h1.weights .+ h2.weights
+    @test (h1 - h2).weights == h1.weights .- h2.weights
+    @test (h1 * h2).weights == h1.weights .* h2.weights
+    @test (h1 / h2).weights == h1.weights ./ h2.weights
+    @test (h1 + 1.5).weights == h1.weights .+ 1.5
```

Comment:
Sorry to raise this only now, but what's the use case for adding a real value to all weights? That doesn't seem to make a lot of sense to me.

Reply:
you may have a "flat background", but ok, this is not terribly useful, I can live without it if you think it does harm